nobrainer/processing/base.py

```diff
@@ -20,7 +20,7 @@ class BaseEstimator:
     state_variables = []
     model_ = None
 
-    def __init__(self, checkpoint_filepath=None, multi_gpu=False):
+    def __init__(self, checkpoint_filepath=None, multi_gpu=True):
```

Comment:
what's the logic of setting this default?

Reply:
I think it makes sense to default to using all available resources.